src/robot/utils/__init__.py

```diff
@@ -74,3 +74,4 @@
                    split_args_from_name_or_path)
 from .unic import prepr, unic
 from .utf8reader import Utf8Reader
+from collections import OrderedDict
```

Comment:
same comment here about PEP8 recommendation for import grouping order

Reply:
No need to import OrderedDict in here.